flexget/plugins/cli/series.py

```diff
@@ -282,8 +284,14 @@ def register_parser_arguments():
     order.add_argument('--descending', dest='order', action='store_true', help='Sort in descending order')
     order.add_argument('--ascending', dest='order', action='store_false', help='Sort in ascending order')
 
-    subparsers.add_parser('show', parents=[series_parser, table_parser],
-                          help='Show the releases FlexGet has seen for a given series ')
+    show_parser = subparsers.add_parser('show', parents=[series_parser, table_parser],
+                          help='Show the releases FlexGet has seen for a given series')
+    show_parser.add_argument('--sort-by', choices=('seen', 'entity'), default='seen',
+                             help='Choose releases list sort: last seen date (default) or entity ID')
+    show_order = show_parser.add_mutually_exclusive_group(required=False)
+    show_order.add_argument('--descending', dest='order', action='store_true', help='Sort in descending order')
+    show_order.add_argument('--ascending', dest='order', action='store_false', help='Sort in ascending order')
+
```

Comment:
`store_true` stores a default of `False` (and the inverse for `store_false`). If neither is specified on the command line, `order` is given the default value of the first `add_argument` call.

```
import argparse
parser = argparse.ArgumentParser()
parser.add_argument('--sort-by', choices=('age', 'entity'), default='age')
order = parser.add_mutually_exclusive_group(required=False)
order.add_argument('--ascending', dest='order', action='store_false')
order.add_argument('--descending', dest='order', action='store_true')

>>> parser.parse_args('--sort-by age'.split())
Namespace(order=True, sort_by='age')
```